starknet_py/tests/e2e/fixtures/contracts_v1.py

```diff
@@ -0,0 +1,164 @@
+# pylint: disable=redefined-outer-name
+from typing import Tuple
+
+import pytest
+import pytest_asyncio
+
+from starknet_py.common import create_casm_class
+from starknet_py.hash.casm_class_hash import compute_casm_class_hash
+from starknet_py.net.account.base_account import BaseAccount
+from starknet_py.net.models import DeclareV2
+from starknet_py.tests.e2e.fixtures.constants import CONTRACTS_COMPILED_V1_DIR, MAX_FEE
+from starknet_py.tests.e2e.fixtures.misc import read_contract
+
+
+async def declare_v1_contract(
```

Comment:
Could we change the names of the functions in this file to use `cairo1` instead of `v1`?
Here for example the function starts with `declare_v1` which kind of conflicts with the declare of version 1.